project.clj

```diff
@@ -3,5 +3,10 @@
   :url "https://github.com/weavejester/suspendable"
   :license {:name "The MIT License"
             :url "http://opensource.org/licenses/MIT"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [com.stuartsierra/component "0.2.3"]])
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [com.stuartsierra/component "0.4.0"]]
+  :profiles {:provided {:dependencies [[org.clojure/clojurescript "1.10.597"]]}}
+  :aliases {"test-cljs" ["with-profile" "provided" "run" "-m" "cljs.main" "-re" "node" "-e"
+                         "(require '[clojure.test :as test])
+                          (require '[suspendable.core-test])
+                          (test/run-all-tests)"]})
```

Comment:
Looks like this is runnign tests unrelated to Suspendable. Perhaps this should be changed to:

```clojure
(test/run-tests 'suspendable.core-test)
```

Reply:
On it